website/docs/examples/next-babel.md

```diff
@@ -1,13 +0,0 @@
----
-sidebar_position: 3
----
-
-# Next with Babel config
-
-## Integrate Jest and Jest Preview to your Next.js app
-
-See more at [Next.js Babel Example](https://github.com/nvh95/jest-preview/tree/main/examples/nextjs-babel)
-
-## Caveats
-
-Next's `<Image />` component has yet to work with Jest Preview.
```

Comment:
This is not mentioned in https://github.com/nvh95/jest-preview/tree/main/examples/nextjs-babel
Please help to move it there.

Reply:
Same 👇

Actually they are there

🚧 Next.js's <Image /> component isn't working with Jest Preview yet.